content/developer/reference/backend/views.rst

```diff
@@ -1931,6 +1933,25 @@ Possible children elements of the list view are:
 
   A special button (`type="edit"`) can be defined to open the many2one form view.
 
+``header``
+  defines custom :ref:`buttons <reference/views/list/button>` in the control panel that perform an action/call a model's method.
+
+  .. code-block:: xml
+
+      <header>
+          <button name=toDo" type="object" string="Always Display" display="always"/>
+          <button name="toDo" type="object" string="Selection Display"/>
+      </header>
+
+  ``button``
+    as :ref:`button <reference/views/list/button>` with:
+
+    ``display``
+      By default, those buttons are only displayed when some records are
+      selected, and they apply on the selection. When the attribute ``display``
+      is set to ``always``, the button is available all the time, even if there's
+      no selection.
```

Comment:
I find it confusing and disruptive (compared to the rest of the page) to duplicate *part of* the reference of `button` inside that of `header` *without* mentioning the existence of the `display` attribute in the original definition of `button`, which is where readers will look for an explanation on `display`.

Now, if you're convinced that it is clearer, please consider adding more text to make it a bit easier to understand:
```suggestion
``header``
  defines custom :ref:`buttons <reference/views/list/button>` in the control panel that perform an action/call a model's method.

  .. code-block:: xml

      <header>
          <button name=toDo" type="object" string="Always Display" display="always"/>
          <button name="toDo" type="object" string="Selection Display"/>
      </header>
   
  Does not support any attribute but can have a child:             

  .. rst-class:: o-definition-list

  ``button``
    as a :ref:`button <reference/views/list/button>` which accepts an extra attribute when placed in a `header`:
    
    .. rst-class:: o-definition-list

    ``display``
      By default, those buttons are only displayed when some records are
      selected, and they apply on the selection. When the attribute ``display``
      is set to ``always``, the button is available all the time, even if there's
      no selection.
```